### PR TITLE
[API-161] Update docs to reflect new field

### PR DESCRIPTION
**Pull request by benmmari**
Jira: https://zigroup.atlassian.net/browse/API-161

Update the API docs to cater for the new field.

Can check it out here -> https://developers.sandbox.zappi.io/

**Screenshots**
screely-1638992389702

screely-1638992406403

**Related PRs:**

1. https://github.com/Intellection/zappistore.app/pull/8648
2. https://github.com/Intellection/zappi-api/pull/103

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Please note that as we are still finalizing our API, this spec is subject to cha
 
 # Change Log
 
+## 7 December 2021
+
+Added `customer_hashed_email_address` to the order payload.
```
Comment (josef-bolt) [inline source/index.html.md:32]: worth mentioning explicitly that it's contained within a metadata object here?

**Review by josef-bolt — COMMENTED** [review]
could you add some screenshots of the changes? I know you added a link - but that will change if we ever deploy over it.

File: source/index.html.md (hunk 1 above)
Comment (benmmari) [reply source/index.html.md:32]: true. lemmy do that

**Comment by benmmari** [comment]
> could you add some screenshots of the changes? I know you added a link - but that will change if we ever deploy over it.

Da done done.

**Review by josef-bolt — APPROVED** [review]
:shipit:

**Pull request merged**